e2e/playwright/secrets.ts

```diff
@@ -15,9 +16,12 @@ try {
     })
 } catch (err) {
   // probably running in CI
-  secrets.token = process.env.token || ''
-  secrets.snapshottoken = process.env.snapshottoken || ''
-  // add more env vars here to make them available in CI
+  console.warn(
+    `Error reading ${secretsPath}; environment variables will be used`
+  )
 }
+secrets.token = secrets.token || process.env.token || ''
```

Comment:
no wait its back!

Reply:
wait while we are here can we just ALL CAPS THE ENV VARS

Reply:
This PR is a side quest for me, somewhat triggered by the security efforts. Can we do it in a separate PR?

Reply:
no worries i get it sorry